schemas/appeal.schema.json

```diff
@@ -0,0 +1,317 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "appeal.schema.json",
+  "title": "Appeal Document",
+  "description": "Schema defining the metadata for an appeal",
+  "type": "object",
+  "additionalProperties": true,
+  "required": [],
+  "properties": {
+    "caseReference": {
+      "type": "string",
+      "examples": ["7002471"],
+      "description": "External case identifier"
+    },
+    "caseStatus": {
+      "type": "string",
+      "enum": [
```

Comment:
Need to add additional statuses for:
- complete
- closed
- awaiting_transfer
- transferred
- withdrawn